src/main/kotlin/com/cosmotech/api/rbac/CsmRbac.kt

```diff
@@ -83,15 +83,29 @@ open class CsmRbac(
     return rbacSecurity
   }
 
+  fun addUserRole(
+      parentRbacSecurity: RbacSecurity,
+      rbacSecurity: RbacSecurity,
+      userId: String,
+      role: String,
+      rolesDefinition: RolesDefinition = getCommonRolesDefinition()
+  ): RbacSecurity {
+    this.getAccessControlOrThrow(
+        parentRbacSecurity,
+        userId,
+        "User '$userId' not found in parent ${parentRbacSecurity.id} component")
```

Comment:
If the purpose of this method is to "check"/"verify" that a user has rights defined in a rbacSecurity, could you please renamed it like `checkRights` or `verifyRights`.
Calling a `getMethod` without using the result is confusing.